### DISCUSSION TITLE
Sobre el Proyecto

### DISCUSSION THREAD

**Discussion by MiryamV** [rramosp/ai4eng.v1#92] [00 - General]
Hola profe.
Quería preguntarle cual de estos temas me recomienda, teniendo en cuenta que casi no manejo lo de programación.

https://www.kaggle.com/c/competitive-data-science-predict-future-sales/data
https://www.kaggle.com/c/house-prices-advanced-regression-techniques/data
https://www.kaggle.com/girardi69/marathon-time-predictions
https://www.kaggle.com/c/titanic
https://www.kaggle.com/c/g-research-crypto-forecasting/data
https://www.kaggle.com/c/petfinder-pawpularity-score/data

Le agradezco.

**Comment by rramosp** [comment]
te comento. los dos que me parecen adecuados son los siguientes, que en el fondo son series temporales:

https://www.kaggle.com/c/competitive-data-science-predict-future-sales/data
https://www.kaggle.com/c/g-research-crypto-forecasting/data

estos dos son demasiado básicos
https://www.kaggle.com/c/house-prices-advanced-regression-techniques/data
https://www.kaggle.com/c/titanic

este es sólo un dataet, no un challenge, con lo que es más dificil saber qué hacer ... además hay pocos competidores y código del que aprender, y el dataset es super pequeño
https://www.kaggle.com/girardi69/marathon-time-predictions

este me parece demasiado complejo, mezclando estadísticas, con imágenes, etc.
https://www.kaggle.com/c/petfinder-pawpularity-score/data
